src/types/ptr_type.rs

```diff
@@ -9,7 +9,6 @@ use crate::values::{ArrayValue, AsValueRef, IntValue, PointerValue};
 use crate::AddressSpace;
 
 use crate::types::enums::BasicMetadataTypeEnum;
```

Comment:
Should const_zero and const_null be removed from PtrType?

Reply:
why would that be? you would now just get a zero/null opaque pointer

Reply:
Ah, I just thought all const fns were removed